### ISSUE TITLE
US is missing state-level data

### ISSUE THREAD

**Issue by pbulises** [CSSEGISandData/COVID-19#1594]
Only data available is at the country level

**Comment by mrwallison** [comment]
Still missing state data as of 03/26/20.  Will state level granularity ever be restored?  It appears that other countries have regional data and the US data was previously available.  What happened?

**Comment by mrwallison** [comment]
If state level detail is not going to be supported please let me know and I will move on.

**Comment by fordmaxson** [comment]
Agree. Please let us know if U.S. state level data will not be supported in the 3 files at
https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series

**Comment by XtremeOwnageDotCom** [comment]
https://github.com/CSSEGISandData/COVID-19/issues/1250

It was removed on purpose.
